src/imap/model.rs

```diff
@@ -50,14 +70,41 @@ impl<'a> ImapConnector<'a> {
         Ok(Self { account, sess })
     }
 
+    /// Closes the connection.
+    ///
+    /// **ALWAYS CALL THIS IF YOU DON'T NEED THE CONNECTION ANYMORE!**
```

Comment:
Sounds like if you don't close the connection, a IMAP fanatic will come to your house and kill all your family :rofl: maybe the bold + capitalize is a bit too much.

Reply:
haha, you're right :sweat_smile:

Reply:
haha, you're right :sweat_smile: